README.md

```diff
@@ -316,6 +316,16 @@ it('should not dispatch correct number of change events', () => {
   expect(onChangeSpy).not.toHaveBeenCalledTimes(2);
 });
 ```
+You can also select `<option>` elements by passing the `HTMLOptionElement`(s) as arguments to `selectOption`. This is particularly useful when you are using `[ngValue]` binding on the `<option>`:
+```ts
+it('should set the correct option on single select when passing the element', () => {
+  const select = spectator.query('#test-single-select-element') as HTMLSelectElement;
+
+  spectator.selectOption(select, spectator.query(byText('Two')) as HTMLOptionElement);
+
+  expect((spectator.query(byText('Two')) as HTMLOptionElement).selected).toBe(true);
```

Comment:
We should be consistent and make the API:

```ts
expect(select).toHaveSelectedOptions(spectator.query(byText('One')));

expect(select).toHaveSelectedOptions([
   spectator.query(byText('One')), 
   spectator.query(byText('Two'))
]);
```

Reply:
Failed test message would need to be like `Expected element to have options '[<option ng-reflect-ng-value="[object Object]">Three</option>]' but had '[<option ng-reflect-ng-value="[object Object]">Two</option>]'`